ui/test-view/ui/LogViewer.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import logUrl from '../../img/logviewerIcon.png';
+
+export class LogViewer extends React.Component {
+    render() {
+        return (
+            <span className="logviewer badge">
+                <Link to={`${SERVICE_DOMAIN}/logviewer.html#/?repo=${this.props.repo}&job_id=${this.props.job.jobId}`}
```

Comment:
Ahh, really good point.  In addition to what you pointed out, this is also more correct because those links would then be referring to, likely, production treeherder.  Where you most likely want it to navigate to the Treeherder UI you're running locally.  Thanks for catching that!